packages/vue-virtual/src/index.ts

```diff
@@ -19,25 +27,25 @@ function useVirtualizerBase<TScrollElement, TItemElement extends Element>(
   const opts = unref(options)
   const virtualizer = new Virtualizer(opts)
   const state = shallowRef(virtualizer)
+  let doClean: (() => void) | undefined
 
-  const applyOptions = (() => {
-    let doClean: (() => void) | undefined
-    return (options: VirtualizerOptions<TScrollElement, TItemElement>) => {
-      doClean?.()
-      virtualizer.setOptions({
-        ...options,
-        onChange: (instance) => {
-          // Force an update event
-          triggerRef(state)
-          options.onChange?.(instance)
-        },
-      })
+  const applyOptions = (
+    options: VirtualizerOptions<TScrollElement, TItemElement>,
+  ) => {
+    doClean?.()
+    virtualizer.setOptions({
+      ...options,
+      onChange: (instance) => {
+        // Force an update event
+        triggerRef(state)
+        options.onChange?.(instance)
+      },
+    })
 
-      virtualizer._willUpdate()
-      virtualizer.measure()
-      doClean = virtualizer._didMount()
-    }
-  })()
+    virtualizer._willUpdate()
+    virtualizer.measure()
```

Comment:
Is this still necessary? There is a `ResizeObserver` watching for the scrollElement size change. I'm not having any issues when the initial size of the scrollElement is at 0 on the first render without it.